deployment/prometheus/config.yml

```diff
@@ -0,0 +1,11 @@
+global:
+  scrape_interval: 5s
+  external_labels:
+    monitor: 'temporal-monitor'
+scrape_configs:
+  - job_name: 'prometheus'
+    metrics_path: /metrics
+    scheme: http
+    static_configs:
+      - targets:
+          - 'temporal:8000'
```

Comment:
This is no longer scraping the worker.

Reply:
what is the host:port worker is pushing to?

Reply:
Prometheus is pull, not push. This is the scrape target for the worker: https://github.com/temporalio/background-checks/pull/19/files#diff-66184e58408c7101ed064f2f630f09031b47b935716fdc2b496b90366f1a3b59L10 Note that it should be just `worker` now as I've removed the `lp` prefixes.

Reply:
cant click on that link somehow. Just see - TEMPORAL_GRPC_ENDPOINT=lp-temporal:7233
not sure if thats what you are talking about